single_kernel_mongo/managers/upgrade.py

```diff
@@ -238,8 +241,10 @@ def run_post_upgrade_checks(self, finished_whole_cluster: bool = False) -> None:
             )
             raise UnhealthyUpgradeError
 
-        if self.charm.unit.status == UNHEALTHY_UPGRADE:
-            self.charm.status_manager.set_and_share_status(ActiveStatus())
+        # TODO this will be addressed in the Advanced Status Handling, when we have the
+        # functionality to clear a status.
```

Comment:
similar comment, is this handled in PR#38?

Reply:
Yes it is